src/app/qgisapp.cpp

```diff
@@ -13684,8 +13685,27 @@ void QgisApp::customProjection()
 
 void QgisApp::newBookmark()
 {
-  showBookmarks( true );
-  mBookMarksDockWidget->addClicked();
+  QString projStr;
+  if ( QgsProject::instance() )
```

Comment:
This will always exist in app

Reply:
actually, as far as i can see, the result of this logic isn't actually used...

To be honest, I'd say it should all be dropped, and we should instead default to an empty group.